gwsurrogate/surrogate.py

```diff
@@ -1888,6 +1893,14 @@ def __call__(self, q, chiA0, chiB0, M=None, dist_mpc=None, f_low=None,
             # Follows the LAL convention (see help text)
             h = self._mode_sum(h, inclination, np.pi/2 - phi_ref,
                     fake_neg_modes=fake_neg_modes)
+        else: # if returning modes, check if m<0 modes need to be generated
+            if (ellMax is not None) and (not self.keywords['Precessing']):
+                modes = list(h.keys())
+                for mode in modes:
+                    ell = mode[0]
+                    m   = mode[1]
+                    if m > 0:
+                        h[(ell,-m)] = (-1)**ell * h[(ell,m)].conjugate()
```

Comment:
I think you need a check for whether this mode already exists. Unlikely, for nonprecessing models, but a potentail way for this code to fail.

Reply:
Done!